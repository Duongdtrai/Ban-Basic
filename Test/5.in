3
35
276 25 420 620 773 291 525 805 208 248 126 663 293 107 994 930 454 786 479 813 267 528 212 640 376 369 76 394 590 401 914 146 607 400 166 
29
609 989 766 775 212 176 74 126 268 123 726 305 41 602 125 467 872 122 390 239 157 35 36 475 393 605 843 553 345 
46
197 887 772 639 937 865 376 185 521 827 778 531 568 902 3 700 703 768 147 107 579 191 812 522 682 91 404 168 596 72 347 14 979 39 935 398 85 483 694 200 230 806 974 50 365 76 
