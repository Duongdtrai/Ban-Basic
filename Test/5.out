655613112
1460979200
6610798180
